channels/rdpdr/client/rdpdr_main.c

```diff
@@ -670,7 +670,7 @@ static UINT handle_hotplug(rdpdrPlugin* rdpdr)
 		if (!path)
 			continue;
 		/* copy hotpluged device mount point to the dev_array */
-		if (isAutomountLocation(path) && (size <= MAX_USB_DEVICES))
+    if (isAutomountLocation(path) && (size < MAX_USB_DEVICES))
```

Comment:
please run `clang-format` to enforce code guidelines.

Reply:
Sorry, I didn't saw that this project use clang-format.
Strange that in Atom everything looks fine.